### ISSUE TITLE
Verison 4.0.3 is not in maven central

### ISSUE THREAD

**Issue by levackt** [web3j/web3j-spring-boot-starter#23]
Getting started guide refers to 4.0.3 which was is not in the repo.

**Comment by konfilios** [comment]
@levackt it looks like an error in the documentation page. Just use 1.6.0 and you'll be fine.

**Comment by iikirilov** [comment]
The version of this project should reflect the version of web3j it is using

**Comment by hd51515** [comment]
does  spring boot web3j starter1.6.0 really use web3j 1.6.0?  what should be done to try a more recent version of web3j like 4.0.3 as my application needs several bug fixes from more recent releases

**Comment by iikirilov** [comment]
No, it doesn't. The decision to synchronize versions in all web3j projects was taken from v.4.

v 4.0.3 of this project is available in jcenter.

@snazha-blkio any reason you decided to upload the artifacts to jcenter only? I can see all previous version of this project are in maven central only.

https://mvnrepository.com/artifact/org.web3j/web3j-spring-boot-starter
